superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/transformProps.ts

```diff
@@ -290,12 +292,20 @@ export default function transformProps(
   const showValueIndexesB = extractShowValueIndexes(rawSeriesB, {
     stack,
   });
+
+  const metricsLabels = metrics
+    .map(metric => getMetricLabel(metric, undefined, undefined, verboseMap))
+    .filter((label): label is string => label !== undefined);
+  const metricsLabelsB = metricsB.map((metric: QueryFormMetric) =>
+    getMetricLabel(metric, undefined, undefined, verboseMap),
```

Comment:
@kgabryje lmk what you think. I don't like having to pass a bunch of undefined values, but it felt safer here than changing the args to kwargs (a js object). A bunch of the map functions are passing values that were unused in getMetricLabel.